widgets/more_buttons.py

```diff
@@ -35,33 +36,35 @@ def add_button(self, text, optionstring):
 
     def add_buttons(self, option = None):
         self._target_obj = obj = option.bound_to if hasattr(option, 'bound_to') else None
-        optionstring = None
+        optionstrings = []
```

Comment:
Nit: Although I can imagine how this change can become necessary in the future, it doesn't feel like it belongs to this work. At the moment only one single button can be shown at a time. I would rather make only the changes that are relevant to the work (easier to review and potentially bisect). And the change could have been made when more than one button needs to be shown, _if ever_.

(But it's a minor thing, so I don't think reverting it is worth the time.)

Reply:
Depending on how area buttons are implemented (if they are), having more than one button would be nice.